docs/src/api/class-electron.md

```diff
@@ -1,7 +1,8 @@
 # class: Electron
 * langs: js
 
-Playwright has **experimental** support for Electron automation. You can access electron namespace via:
+Playwright has **experimental** support for Electron automation. Minimal supported Electron version is **12**.
```

Comment:
See here: https://github.com/microsoft/playwright/pull/8759#issuecomment-933548946

FTR, not sure if we want to mention them all tho.

Reply:
Updated